README.md

```diff
@@ -284,7 +284,7 @@ my_favorite_prior = Prior('Laplace', mu=0., b=10)
 
 # Set the prior when adding a term to the model;
 # more details on this below.
-priors = {'subject': my_favorite_prior}
+priors = {'1|subject': my_favorite_prior}
```

Comment:
I can't remember, is the previous syntax now actually erroneous, or just no longer the preferred syntax?

Reply:
It errors, because it can't find a term with that key. We could probably change it to do more clever matching, but I think strictness is fine here--we no longer allow passing just `'subject'` as a term to `random` (i.e., has to be `1|random`), so I think we may as well keep it like this.

I can no longer replicate the error I mentioned above. It's possible it was actually due to the `autoreload` magic doing wonky things, and not to a problem with the code. So feel free to merge once you've reviewed the PR.